refactoring.php

```diff
@@ -2,50 +2,49 @@
 
 public function post_confim() {
     $id = Input::get('service_id');
-    $servicio = Service::find($id);
-    //dd($servicio)
-    if ($servicio != NULL) {
-        if ($servicio->status_id == '6') {
-            return Response::json(array('error' => '2'));
-        }
-        if ($servicio->driver_id == NULL && $servicio->status_id == '1') {
-            $servicios = Service::update($id, array(
-                         'driver_id' => Input::get('driver_id'),
-                         'status_id' => '2'
-                            //Up Carro
-                            //,'pwd' => md5(Input::get('pwd'))
-            ));
-            Driver::update(Input::get('driver_id'), array(
```

Comment:
Avoid call the same function / method many time `Input::get('driver_id')` better call once and store the return value on a variable and used this variable